portal-web/test/functional/com/liferay/portalweb/macros/User.macro

```diff
@@ -2237,28 +2237,16 @@ definition {
 
 	@description = "Log out of Liferay Portal."
 	macro logoutPG {
-		if (isSet(nodePort)) {
-			Navigator.openSpecificURL(url = "http://localhost:${nodePort}");
+		if (isSet(specificURL)) {
+			Navigator.openSpecificURL(url = "${specificURL}");
```

Comment:
Simple the condition, so it's less different with the [logoutUserPG ](https://github.com/Sharryxue/liferay-portal/pull/254/files#diff-ddf0f6ac34b48653aae4320acee8285c0b63a3a83b2ac227172f7719e491c06eL2255-L2261)macro below, and we can combine them.